hasek/config.py

```diff
@@ -0,0 +1,405 @@
+import os
+import sys
+
+import yaml
```

Comment:
Not a necessary thing, but I wonder if it would make sense to use a structured file format that's supported as a built-in (ini w/configparser? pretty-printed json?), instead of relying on yaml - if the idea is that you should only be interacting with the `.hasekrc` file through the command line tool, the readability of yaml is not as critical.  Could cut down a dependency.